src/blocks/stars.css

```diff
@@ -0,0 +1,41 @@
+.card__container {
```

Comment:
why did you place bem elements of card inside stars.css file?

Reply:
you should create files only for bem blocks

create a separate bem block for stars